<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>counterpartCardPage</class>
 <widget class="ma::chrono::CardPage" name="counterpartCardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> - Counterparts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="contentsTab">
      <attribute name="title">
       <string>Contents</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLineEdit" name="fullNameEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="fullLatinNameEdit"/>
       </item>
       <item row="9" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="shortNameEdit"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="shortLatinNameEdit"/>
       </item>
       <item row="8" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="countryEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="countryBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Full name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fullNameAsterikLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>required field</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;/&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;span style=&quot; color:red;&quot;&gt;*&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Full latin name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fullLatinNameAsterikLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>required field</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;/&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;span style=&quot; color:red;&quot;&gt;*&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Short name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="shortNameAsterikLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>required field</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;/&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;span style=&quot; color:red;&quot;&gt;*&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Short latin name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="shortLatinAsterikLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>required field</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;/&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;span style=&quot; color:red;&quot;&gt;*&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="countryAsterikLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>required field</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;/&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;span style=&quot; color:red;&quot;&gt;*&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>User, creator</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="createUserEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="createUserBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Time, created</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="createTimeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>User, last update</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="updateUserEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="updateUserBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Time, last update</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="updateTimeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>358</width>
           <height>42</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ma::chrono::CardPage</class>
   <extends>QWidget</extends>
   <header location="global">ma/chrono/cardpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>fullNameEdit</tabstop>
  <tabstop>fullLatinNameEdit</tabstop>
  <tabstop>shortNameEdit</tabstop>
  <tabstop>shortLatinNameEdit</tabstop>
  <tabstop>countryEdit</tabstop>
  <tabstop>countryBtn</tabstop>
  <tabstop>createUserEdit</tabstop>
  <tabstop>createUserBtn</tabstop>
  <tabstop>createTimeEdit</tabstop>
  <tabstop>updateUserEdit</tabstop>
  <tabstop>updateUserBtn</tabstop>
  <tabstop>updateTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
